static/js/containers/SearchPage.js

```diff
@@ -271,6 +263,7 @@ const mapStateToProps = (state, ownProps) => {
     ? channels.error && channels.error.errorStatusCode === 403
     : false
   const { results, total, initialLoad } = search.data
+  const upvotedPosts = posts.data
```

Comment:
I just noticed that this change has a nice side effect, which is that the `posts` reducer also holds any posts which the user has already loaded from the frontpage or channel APIs. So if I've upvoted a post, and it was already returned by the frontpage or the channel page API, then I'll see accurate information about whether it's upvoted when I look at the same post in search. Nice!